cmd/kubelet/app/server.go

```diff
@@ -423,9 +424,13 @@ func Run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 	return nil
 }
 
+// checkPermissions confirms that, on linux, the UID of the kubelet process is 0.  For windows, it will return an error automatically.  For other systems, the behaviour is unspecified.
 func checkPermissions() error {
+	if runtime.GOOS == "windows" {
+		return fmt.Errorf("Not checking UID permissions because we're on windows")
+	}
 	if uid := os.Getuid(); uid != 0 {
-		return fmt.Errorf("kubelet needs to run as uid `0`. It is being run as %d", uid)
+		return fmt.Errorf("on linux, the kubelet needs to run as uid `0`, but it is being run as %d", uid)
```

Comment:
According to https://golang.org/pkg/os/#Getuid the function returns -1 on Windows.
But the function behavior is not written on that for the other operating systems.
I feel we don't need to change this error message for now.

Reply:
fixed to the original message.  will think more about it .  im not sure what other OS's do, but ill poke around later